plugins/modules/network_settings_workflow_manager.py

```diff
@@ -1954,9 +1994,15 @@ def get_want_network(self, network_management_details):
 
             primary_server_address = network_aaa.get("primary_server_address")
             if primary_server_address:
-                want_network_settings.get("network_aaa").update({
-                    "network": primary_server_address
-                })
+                if self.is_server_exists(primary_server_address):
```

Comment:
As we are returning for failure, we can simply write below.. We don't need else blocks.. Instead of **in the system**, can we say **in the network**?
```
                if not self.is_server_exists(primary_server_address):
                    self.msg = "The 'primary_server_address' - '{0}' under 'network_aaa' is not found in the system." \
                               .format(primary_server_address)
                    self.status = "failed"
                    return self
                
           want_network_settings.get("network_aaa").update({
                        "network": primary_server_address
                    })

```
Can you do the same for all the below cases?